release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -2758,6 +2758,12 @@ $ ./openshift-install wait-for install-complete
 
 * It is not possible to create a macvlan on the physical function (PF) when a virtual function (VF) already exists. This issue affects the Intel E810 NIC. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2120585[*BZ#2120585*])
 
+* There is currently a known issue when manually configuring IPv6 addresses and routes on an IPv4 {product-title} cluster when converting to a dual-stack cluster. Newly created pods become stuck in the `ContainerCreating` status. Currently, there is no workaround. This issue is planned to be addressed in a future {product-title} release. (link:https://issues.redhat.com/browse/OCPBUGS-4411[*OCPBUGS-4411*])
+
+* When an OVN cluster has more than 75 worker nodes, simultaneously creating 2000 or more services and route objects can cause pods created at the same time to hang in the `ContainerCreating` status. If this problem occurs, entering the `oc describe pod <podname>` command will show events with the following warning: `FailedCreatePodSandBox...failed to configure pod interface: timed out waiting for OVS port binding (ovn-installed)`. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGS-3470[*OCPBUGS-3470*])
```

Comment:
Use "remains" instead of "hang": "...can cause pods created at the same time to remain in the ...."

Use present tense: "...command shows events with...